src/lightning/fabric/connector.py

```diff
@@ -106,7 +106,7 @@ def __init__(
         strategy: Union[str, Strategy] = "auto",
         devices: Union[List[int], str, int] = "auto",
         num_nodes: int = 1,
-        precision: _PRECISION_INPUT = "32-true",
+        precision: Optional[_PRECISION_INPUT] = None,
```

Comment:
What is the default behavior/precision?

Reply:
32-true. This is mentioned in the Fabric and Trainer docstrings